public/index.html

```diff
@@ -13,36 +13,36 @@
       crossorigin="anonymous"
       referrerpolicy="no-referrer"
     />
-    <link
-      rel="stylesheet"
-      href="https://site-assets.fontawesome.com/releases/v6.1.1/css/all.css"
-    />
-  </head>
-  <body>
+    <link rel="stylesheet" href="https://site-assets.fontawesome.com/releases/v6.1.1/css/all.css">
+
+</head>
+<body>
     <header>
-      <div class="container">
-        <div class="leftside">
-          <i class="fa-solid fa-dog"></i>
-          <h2>Doggie Pics</h2>
-        </div>
-        <div class="rightside">
-          <h2><a href="#">HOME</a></h2>
```

Comment:
apparently you haven't pulled the main branch before making your changes, try to restore the parts that you weren't working on to their original state before adding your changes to prevent further conflict

Reply:
Already up to date.

Reply:
h2 tags here were reverted to p in your version.